test.lua

```diff
@@ -241,7 +243,7 @@ local cases = {
         ["(let [(a b c d e f g) (if (= (+ 1 1) 2) (values 1 2 3 4 5 6 7))] (+ a b c d e f g))"]=28,
         -- IIFE in if statement required v2
         ["(let [(a b c d e f g) (if (= (+ 1 1) 3) nil\
-                                       ((or unpack table.unpack) [1 2 3 4 5 6 7]))]\
+                                       ((or _G.unpack table.unpack) [1 2 3 4 5 6 7]))]\
```

Comment:
What was the rationale for adding the `_G` here? Not a big deal, it just doesn't look necessary since this test is running with the default `allowedGlobals`.

Reply:
Same as above; this is now needed because we removed `allowGlobals = false` from the eval call that the tests run under.